sbol3/boolean_property.py

```diff
@@ -0,0 +1,41 @@
+from typing import Union, Any, List, Optional
+
+import rdflib
+
+from . import *
+
+
+class BooleanPropertyMixin:
+
+    def from_user(self, value: Any) -> Union[None, rdflib.Literal]:
+        if value is None:
+            return None
+        if not isinstance(value, bool):
+            raise TypeError(f'Expecting boolean, got {type(value)}')
+        return rdflib.Literal(value)
+
+    def to_user(self, value: Any) -> int:
```

Comment:
Shouldn't this be `-> bool` since it returns a bool?

Reply:
Yep.  I fixed it.